mne/utils.py

```diff
@@ -510,6 +510,21 @@ def _reject_data_segments(data, reject, flat, decim, info, tstep):
     return data, drop_inds
 
 
+def _get_inst_data(inst):
```

Comment:
I'll add support for other mne objects in a separate PR.
Regarding `copy` param - I think in this case it is straightforward to just do `_get_inst_data(inst).copy()`.